job/replay.go

```diff
@@ -0,0 +1,187 @@
+package job
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/odpf/optimus/core/cron"
+	"github.com/odpf/optimus/core/multi_root_tree"
+	"github.com/odpf/optimus/models"
+	"github.com/pkg/errors"
+)
+
+const (
+	// DateFormat YYYY-mm-dd for replay dates and dag start date
+	DateFormat = "2006-01-02"
```

Comment:
Can we prefix it with `Replay` so that it's clear it's for replay only?

Reply:
updated